js-extensions/spec/ExtensionRenderer-spec.js

```diff
@@ -27,8 +27,8 @@ const mockExtensionStore = {
 };
 
 const mockResourceLoadTracker = {
-    onMount: function() {
-        // Don't care
+    onMount: function(_, onload) {
```

Comment:
`_` is not used and what is that any way?

Reply:
makes lint happy